server.js

```diff
@@ -50,28 +50,26 @@ app.get("/email", async (req, res) => {
 
 
 app.get("/openai", async (req, res) => {
-  console.log(req.query);
+  
+  const {recipe_country_of_origin, is_lactose_intolerant, is_vegan, what_are_user_other_dietary_requirements } = req.query
```

Comment:
I see what you've done here. It's a noticeable improvement! Great work @josueJURE